test/jest/utils.ts

```diff
@@ -37,11 +37,11 @@ export class MockForwarder {
         this.moduleId = id || 1;
     }
 
-    public register = (config: SDKInitConfig): void => {
+    public register = (config: KitRegistrationConfig): void => {
         if (config.kits) {
             config.kits[this.name] = {
-                constructor: this.constructor,
-            };
+                constructor: this.constructor as () => RegisteredKit,
```

Comment:
is this line necessary?  i would have expected just lin 44 to be the `RegisteredKit`

Reply:
It is if we make `filters` required. By making it optional, I can get rid of line 44.